pkg/diagnostics/pod/dns.go

```diff
@@ -101,6 +105,7 @@ func resolveSearch(resolvConf *dns.ClientConfig, r types.DiagnosticResult) {
 		return string(b)
 	}()
 	for _, domain := range resolvConf.Search {
+		// TODO: use dnsDomain instead of cluster.local
```

Comment:
I meant that.

On Feb 18, 2016, at 7:47 PM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/diagnostics/pod/dns.go
<https://github.com/openshift/origin/pull/7446#discussion_r53408001>:

> @@ -101,6 +105,7 @@ func resolveSearch(resolvConf *dns.ClientConfig, r types.DiagnosticResult) {
>  		return string(b)
>  	}()
>  	for _, domain := range resolvConf.Search {
> +		// TODO: use dnsDomain instead of cluster.local

kubernetes.default.svc?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/7446/files#r53408001>.